scripts/install_hooks.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Is this script absolutely necessary? If so, I think instructions should be added to the `Contributing` section of the readme

Reply:
hmm we don't really need a new script for this, I've made some changes in the readme on how to enable the hooks